apps/snitch_core/priv/repo/seed/orders.ex

```diff
@@ -31,11 +31,22 @@ defmodule Snitch.Seed.Orders do
     updated_at: DateTime.utc()
   }
 
+  defp address_order_address(address) do
```

Comment:
That's a very funny name :slightly_smiling_face:

Reply:
any rename suggestions?

Reply:
Naming is hard, and since the body is just 2 lines, move them to the pipe at L48. :metal:
You can even consider `Map.drop/2`.